File: app/src/main/java/com/booboot/vndbandroid/util/image/BlurIfDemoTransform.java
package com.booboot.vndbandroid.util.image;

import android.content.Context;
import android.graphics.Bitmap;

import com.squareup.picasso.Transformation;

public class BlurIfDemoTransform implements Transformation {
    private final static boolean DEMO = false;
    private Context context;

    public BlurIfDemoTransform(Context context) {
        super();
        this.context = context;
    }

    @Override
    public Bitmap transform(Bitmap bitmap) {
        if (!DEMO) return bitmap;
        Bitmap res = BitmapTransformation.darkBlur(context, bitmap);
        bitmap.recycle();
        return res;
    }

    @Override
    public String key() {
        return "darkblur";
    }
}

File: app/src/main/java/com/booboot/vndbandroid/util/image/BlurIfDemoTransform.kt
package com.booboot.vndbandroid.util.image

import android.content.Context
import android.graphics.Bitmap

import com.squareup.picasso.Transformation

class BlurIfDemoTransform(private val context: Context) : Transformation {
    override fun transform(bitmap: Bitmap): Bitmap {
        if (!DEMO) return bitmap
        val res = BitmapTransformation.darkBlur(context, bitmap)
        bitmap.recycle()
        return res
    }

    override fun key(): String {
        return "darkblur"
    }

    companion object {
        private val DEMO = false
    }
}